Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue11703.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.ManualReview)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 11703, "Android textAllCaps no longer works", PlatformAffected.Android)]
+	public class Issue11703 : TestContentPage // or TestMasterDetailPage, etc ...
```

Comment:
You can remove this comment: `// or TestMasterDetailPage, etc ...`

Reply:
removed, thanks @hartez!